Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:16.04
```

Comment:
You can use `ruby:2.4` as a base image and skip ruby and bundler installation steps